ecs_fargate/README.md

```diff
@@ -198,11 +198,15 @@ Configure the AWS FireLens integration built on Datadog's Fluent Bit output plug
             "dd_source": "redis",
             "dd_tags": "project:fluentbit",
             "TLS": "on",
+	     "Host": "http-intake.logs.datadoghq.com",
```

Comment:
I think this is the default value in the [plugin](https://github.com/fluent/fluent-bit/blob/master/plugins/out_datadog/datadog.h#L27). so maybe it is not useful to add it here.

Reply:
Adding explicitly the default value would reduce friction for people in EU site. Allowing them to easily spot and change the endpoint if they want to forward their logs to DD EU.
I would suggest we keep it <3